### PR TITLE
Telcodocs 43

### PR DESCRIPTION
**Pull request by ogradyp**
JIRA
https://issues.redhat.com/browse/TELCODOCS-43 Release Note content for the standalone Node Maintenance Operator

Preview
https://deploy-preview-42404--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-node-maintenance-operator

Versions:
Applies to OpenShift version : 4.10

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -602,6 +602,11 @@ You can now enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgrou
 
 You can enable swap memory use for {product-title} workloads on a per-node basis. For more information, see xref:../nodes/nodes/nodes-nodes-working.adoc#nodes-nodes-swap-memory_nodes-nodes-working[Enabling swap memory use on nodes].
 
+[id="ocp-4-10-node-maintenance-operator"]
+==== Place nodes into maintenance mode using the Node Maintenance Operator
```
Comment (kalexand-rh) [inline release_notes/ocp-4-10-release-notes.adoc:606]:
```suggestion
==== Place nodes into maintenance mode by using the Node Maintenance Operator
```

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -602,6 +602,11 @@ You can now enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgrou
...
+
+The Node Maintenance Operator (NMO) cordons off the node from the rest of the cluster, and drains all the pods from the node. By placing nodes under maintenance, you can investigate problems with a machine, or perform operations on the underlying machine, that might result in a node failure. This is a standalone version of NMO. If you have installed OpenShift Virtualization, then you must use the NMO which is bundled with it.
```
Comment (kalexand-rh) [inline release_notes/ocp-4-10-release-notes.adoc:608]:
```suggestion
The Node Maintenance Operator (NMO) cordons off nodes from the rest of the cluster and drains all the pods from the nodes. By placing nodes under maintenance, you can investigate problems with a machine, or perform operations on the underlying machine, that might result in a node failure. This is a standalone version of NMO. If you installed OpenShift Virtualization, then you must use the NMO that is bundled with it.
```

**Review by kalexand-rh — COMMENTED** [review]
I have few style suggestions, but this looks good!

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (ogradyp) [reply release_notes/ocp-4-10-release-notes.adoc]: Comment included

File: release_notes/ocp-4-10-release-notes.adoc (hunk 2 above)
Comment (ogradyp) [reply release_notes/ocp-4-10-release-notes.adoc]: Comments included

Comment (ogradyp) [reply release_notes/ocp-4-10-release-notes.adoc]: Thanks for your review @kalexand-rh . All comments are included and the PR updated ready for merging

Comment (abhatt-rh) [reply release_notes/ocp-4-10-release-notes.adoc]: @ogradyp Before requesting to merge this, please get it reviewed from @anna-savina and @razo7

**Comment by ogradyp** [comment]
Hi @razo7 / @anna-savina , 
Can you please review this PR from an SME and QE perspective?
Thanks
Padraig

**Comment by ogradyp** [comment]
> /lgtm

**Pull request merged**